app/views/ht_institutions/show.html.erb

```diff
@@ -1,28 +1,35 @@
+<script>
+$(function () {
+  $('[data-toggle="tooltip"]').tooltip()
+})
+</script>
+
 <div id="maincontent" class="row">
   <h1><%= @institution.name %></h1>
 
   <%= render 'shared/flash_message' %>
 
-  <div class="col-sm-6">
-    <dl class="dl-horizontal">
```

Comment:
Removing dl-horizontal seems to result in stacking the dt tags on top of the dd tags, which I don't think is the aim (I can attach a screenshot if we're seeing something different layout-wise). Additionally, when I run HTML validation it looks like the divs inside the dl make the parser unhappy.

Reply:
I think I committed something I didn't intend to. Thanks for bearing with me...

Reply:
Force pushed a corrected commit; this should be fixed now